tests/components/blebox/test_air_quality.py

```diff
@@ -0,0 +1,94 @@
+"""Blebox air_quality tests."""
+
+import logging
+
+import blebox_uniapi
+import pytest
+
+from homeassistant.components.air_quality import ATTR_PM_0_1, ATTR_PM_2_5, ATTR_PM_10
+from homeassistant.const import ATTR_ICON, STATE_UNKNOWN
+
+from .conftest import async_setup_entity, mock_feature
+
+from tests.async_mock import AsyncMock, PropertyMock
+
+
+@pytest.fixture(name="airsensor")
+def airsensor_fixture():
+    """Return a default air quality fixture."""
+    feature = mock_feature(
+        "air_qualities",
+        blebox_uniapi.air_quality.AirQuality,
+        unique_id="BleBox-airSensor-1afe34db9437-0.air",
+        full_name="airSensor-0.air",
+        device_class=None,
+        pm1=None,
+        pm2_5=None,
+        pm10=None,
+    )
+    product = feature.product
+    type(product).name = PropertyMock(return_value="My air sensor")
+    type(product).model = PropertyMock(return_value="airSensor")
+    return (feature, "air_quality.airsensor_0_air")
+
+
+async def test_init(airsensor, hass, config):
+    """Test airSensor default state."""
+
+    _, entity_id = airsensor
+    entry = await async_setup_entity(hass, config, entity_id)
+    assert entry.unique_id == "BleBox-airSensor-1afe34db9437-0.air"
+
+    state = hass.states.get(entity_id)
+    assert state.name == "airSensor-0.air"
+
+    assert ATTR_PM_0_1 not in state.attributes
+    assert ATTR_PM_2_5 not in state.attributes
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5535891)